appsso-starter-java/accelerator.yaml

```diff
@@ -11,6 +11,12 @@ accelerator:
   imports:
     - name: tap-workload
     - name: build-wrapper-gradle
+  options:
+    - name: offeringName
+      description: To find the available offerings in your cluster, run `tanzu services classes list`.
+      inputType: text
+      label: Offering name
+      required: false
```

Comment:
**[question]:** Shouldn't this be required?

Reply:
Yes, you're correct. Updating.